.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,79 @@
+# For most projects, this workflow file will not need changing; you simply need
```

Comment:
I thought you had removed this? Or this is just not activated in the github interface?